azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/VideoViewManager.kt

```diff
@@ -6,26 +6,36 @@ package com.azure.android.communication.ui.presentation
 import android.content.Context
 import android.view.View
 import android.view.ViewGroup
-import com.azure.android.communication.calling.CreateViewOptions
-import com.azure.android.communication.calling.MediaStreamType
 import com.azure.android.communication.calling.RemoteVideoStream
-import com.azure.android.communication.calling.ScalingMode
 import com.azure.android.communication.calling.VideoStreamRenderer
 import com.azure.android.communication.calling.VideoStreamRendererView
+import com.azure.android.communication.calling.MediaStreamType
+import com.azure.android.communication.calling.ScalingMode
+import com.azure.android.communication.calling.CreateViewOptions
 import com.azure.android.communication.ui.service.calling.sdk.CallingSDKWrapper
 
 internal class VideoViewManager(
     private val callingSDKWrapper: CallingSDKWrapper,
     private val context: Context,
 ) {
+    private val remoteParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
+    private val localParticipantVideoRendererMap: HashMap<String, VideoRenderer> = HashMap()
+
     class VideoRenderer constructor(
```

Comment:
constructor keyword not needed.  Also, this class is used only in this file, correct?  If so can we prefix with private